nemo/collections/common/data/lhotse/dataloader.py

```diff
@@ -325,14 +326,15 @@ def get_lhotse_dataloader_from_config(
         )
 
     # 4. Creating dataloader.
-    if is_tarred:
+    if is_tarred and not config.tarred_random_access:
```

Comment:
Remind me - is the IPL dataloader shared between DDP ranks, or each DDP rank has its own?

Reply:
Each rank has its own dataloader

Reply:
in that case make sure you are initializing the IPL dataloader with rank=0 and world_size=1 (maybe you already are - just making sure bugs don't slip through)